### PR TITLE
Add instructions about running tests in IJ w/ alpn agent

### PR DESCRIPTION
**Pull request by lukaszx0**
Copied instructions by @dapengzhang0 posted in #2728

cc @ejona86

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,3 +38,15 @@ Make sure that `./gradlew build` (`gradlew build` on Windows) completes
...
+
+### Jetty ALPN setup for IntelliJ
+
+Some tests require jetty-alpn agent running in the background otherwise they'll fail.
+Here are instructions on how to setup IntellJ IDEA to enable running those tests in IDE:
+
+* Settings -> Build Tools -> Gradle -> Runner -> select Gradle Test Runner
+* View -> Tool Windows -> Gradle -> Edit Run Configuration -> Defaults -> JUnit -> Before lauch -> + -> Run Gradle task, enter the task in the build.gradle that sets the javaagent.
+
+Sept 1 must be taken, otherwise by the default JUnit Test Runner running a single test in IDE will trigger all the tests.
```
Comment (ejona86) [inline CONTRIBUTING.md]: s/Sept/Step/

**Comment by ejona86** [comment]
With #2752 this is now only necessary for the interop-testing project, for the TlsTest and Http2OkHttpTest.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lukaszx0) [inline CONTRIBUTING.md]: Fixed.

**Comment by lukaszx0** [comment]
> With #2752 this is now only necessary for the interop-testing project, for the TlsTest and Http2OkHttpTest.

Awesome, thank you! I've mentioned that it applies to interop-testing explicitly.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lukaszx0) [inline CONTRIBUTING.md]: Fixed.

**Comment by lukaszx0** [comment]
:shipit: ?

**Comment by carl-mastrangelo** [comment]
@lukaszx0 I updated the commit message to be in `package: thing-that-changed` format

**Comment by lukaszx0** [comment]
@carl-mastrangelo thanks, I use this convention but wasn't sure if tagging was required in particular this case. By the way, I figured it would be nice to record those conventions - https://github.com/grpc/grpc-java/pull/2780

**Pull request merged**